lib/easy.c

```diff
@@ -84,6 +84,31 @@
 #include "curl_memory.h"
 #include "memdebug.h"
 
+
+/* curl_decode_data_blob is used by curl library to
+ *  check if a curl parameter is a memory blob instead filename
+ * it return 1 and fill len and data if this is memory blob
+ * and return 0 if this is a filename (or another string) */
+bool curl_decode_data_blob(char *str, size_t *len, void **data)
```

Comment:
Sure this function isn't a public API? If it isn't public it must not start with `curl_`. If it needs to be global within the lib it should be prefixed with `Curl_` (uppercase C).

And what's to "decode" ? The struct points to the data and it contains the size. Why not just use those directly?

Reply:
I rename currenlty to Curl_use_data_blob (and pehaps better name later)